src/common.js

```diff
@@ -67,6 +71,10 @@ var ShellString = function (str, stderr) {
   var that = new String(str);
   that.to = wrap('to', _to, {idx: 1});
   that.toEnd = wrap('toEnd', _toEnd, {idx: 1});
+  that.cat = wrap('cat', _cat, {idx: 1});
```

Comment:
In terms of performance, it actually doesn't hurt much. `wrap()` is actually very quick, since it just returns another function. But I agree with the DRY part, so this should be fixed now.